examples/resources/snowflake_grant_role/import.sh

```diff
@@ -0,0 +1,2 @@
+# format is role_name (string) | grantee_object_type (ROLE|USER) | grantee_name (string)
```

Comment:
Hmm, the docs were not generated for this new resource (the check did not fail btw, we should check why).

Reply:
ran `make docs`